src/features/gcode/GCodeGenerator.js

```diff
@@ -184,7 +183,7 @@ class GCodeGenerator extends Component {
 
         // This loads up (start, end].
         for (let step = 0; step < (delta.magnitude() / maxLength) ; step++) {
-          subsampledVertices.push(Vertex(start.x + step * deltaSegment.x,
+          subsampledVertices.push(new Victor(start.x + step * deltaSegment.x,
                                          start.y + step * deltaSegment.y,
                                          this.props.vertices[next].f))
```

Comment:
I think we can drop these f variables.

Reply:
ah, yes. I missed this one (and maybe others) in the Victor replacement.